arango/batch.py

```diff
@@ -102,12 +122,22 @@ def commit(self):
         for later retrieval via its :func:`arango.batch.BatchJob.result`
         method
 
+        :returns: list of arango.batch.BatchJob or None
+
```

Comment:
Missing the return type in the docstring. Please add:
```python
:rtype: [arango.batch.BatchJob] | None
```

Reply:
yep